corehq/apps/app_manager/models.py

```diff
@@ -2323,6 +2324,8 @@ class CaseList(IndexedSchema, NavMenuItemMediaMixin):
     def rename_lang(self, old_lang, new_lang):
         _rename_key(self.label, old_lang, new_lang)
 
+    def get_app(self):
```

Comment:
Now that every class implementing NavMenuItemMediaMixin has a get_app method, could you add a get_app to NavMenuItemMediaMixin that just raises an exception, in case an implementing class forgets to include get_app?

Reply:
oh yes, sure can do.